reach

```diff
@@ -1209,6 +1397,7 @@ do_usage () {
   echo "  down    --- halt any dockerized devnets for this app"
   echo "  scaffold -- set up Docker scaffolding for a simple app"
   echo "  react   --- run a simple react app"
+  echo "  tls     --- configure TLS certificates for \`reach server\`"
```

Comment:
This command needs to be documented. Ideally, with a two-way link to `reach server`